easybuild/easyconfigs/e/Elmer/Elmer-9.0-foss-2022b.eb

```diff
@@ -0,0 +1,53 @@
+easyblock = 'CMakeMake'
+
+name = 'Elmer'
+version = '9.0'
+
+homepage = 'https://www.csc.fi/web/elmer/elmer'
+description = """Elmer is an open source multiphysical simulation software mainly developed by 
+CSC - IT Center for Science (CSC). Elmer includes physical models of fluid dynamics, structural 
+mechanics, electromagnetics, heat transfer and acoustics, for example. These are described by 
+partial differential equations which Elmer solves by the Finite Element Method (FEM)."""
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+toolchainopts = {'opt': True, 'optarch': True}
+
+source_urls = ['https://github.com/ElmerCSC/elmerfem/archive/']
+sources = ['release-%(version)s.tar.gz']
+patches = ['Elmer-9.0-DCRComplexSolve.patch', 'Elmer-9.0-Qwt-6.2-Fix.patch', 'Elmer-9.0-renderarea-Qt5.patch']
+checksums = [
+    {'release-9.0.tar.gz': '08c5bf261e87ff37456c1aa0372db3c83efabe4473ea3ea0b8ec66f5944d1aa0'},
+    {'Elmer-9.0-DCRComplexSolve.patch': '360d39251cd09423e4caa7cd7b150817fa17724c5eabc00b5427b113de9e4708'},
+    {'Elmer-9.0-Qwt-6.2-Fix.patch': '9abf73537ee1053bd3e49c55fa19b83043ba51fb4a4176600964da0b5cd5a9ca'},
+    {'Elmer-9.0-renderarea-Qt5.patch': '745b98335b5ba43bef7a4ee7123b00bdb4f784c875d138a16dea1d0797567cd9'},
+]
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+]
+
+dependencies = [
+    ('Hypre', '2.27.0'),
+    ('MUMPS', '5.6.1', '-metis'),
+    ('Qt5', '5.15.7'),
+    ('Qwt', '6.2.0'),
+    #    ('Trilinos', '12.0.1', '-Python-2.7.9'),
+]
+
+configopts = '-DWITH_ELMERGUI:BOOL=TRUE -DWITH_QT5:BOOL=TRUE '
+configopts += '-DWITH_MPI:BOOL=TRUE '
+configopts += '-DCMAKE_C_COMPILER=$(which $CC) '
+configopts += '-DCMAKE_Fortran_COMPILER=$(which $F90) '
+configopts += '-DCMAKE_CXX_COMPILER=$(which $CXX) '
+configopts += '-DBLAS_LIBRARIES="-L$EBVARBLAS_LAPACK_INC_DIR $EBVARLIBBLAS" '
+configopts += '-DLAPACK_LIBRARIES="-L$EBVARBLAS_LAPACK_INC_DIR $EBVARLIBLAPACK" '
+configopts += '-DSCALAPACK_LIBRARIES="-L$EBVARSCALAPACK_LIB_DIR $EBVARLIBSCALAPACK" '
+
+sanity_check_paths = {
+    'files': ['lib/elmersolver/libelmersolver.so'] +
+             ['bin/%s' % x for x in ['elmerf90', 'ElmerGrid', 'elmerld', 'ElmerSolver',
+              'ElmerSolver_mpi', 'GebhardtFactors', 'matc', 'Mesh2D', 'ViewFactors']],
+    'dirs': ['share/elmersolver/include']
+}
+
```

Comment:
Are there any `sanity_check_commands` (https://docs.easybuild.io/writing-easyconfig-files/#sanity-check) that you could specify? Something like a `-v` or `--help` or something, just to make sure the binaries in `bin` can start?

Reply:
Added check command.